File: go-client-sdk/pkg/models/shared/codatdatacontractsdatasetscommercelocation.go
package shared

import (
	"time"
)

type CodatDataContractsDatasetsCommerceLocation struct {
	Address            *CodatDataContractsDatasetsCommerceAddress `json:"address,omitempty"`
	ID                 *string                                    `json:"id,omitempty"`
	ModifiedDate       *time.Time                                 `json:"modifiedDate,omitempty"`
	Name               *string                                    `json:"name,omitempty"`
	SourceModifiedDate *time.Time                                 `json:"sourceModifiedDate,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/codatdatacontractsdatasetscommercelocation.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { CodatDataContractsDatasetsCommerceAddress } from "./codatdatacontractsdatasetscommerceaddress";



export class CodatDataContractsDatasetsCommerceLocation extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=address" })
  address?: CodatDataContractsDatasetsCommerceAddress;

  @SpeakeasyMetadata({ data: "json, name=id" })
  id?: string;

  @SpeakeasyMetadata({ data: "json, name=modifiedDate" })
  modifiedDate?: Date;

  @SpeakeasyMetadata({ data: "json, name=name" })
  name?: string;

  @SpeakeasyMetadata({ data: "json, name=sourceModifiedDate" })
  sourceModifiedDate?: Date;
}
